File: 0001-two-sum/0001-two-sum.go
func twoSum(nums []int, target int) []int {

    res := []int{}

	for i, num := range nums {
		diff := target - num

		for x, part := range nums {

			if i == x {
				continue
			}

			if part == diff {
				res = append(res, i, x)
				break
			}
		}

		if len(res) == 2 {
			break
		}
	}

	return res
}

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let twoKeys = []

    for (i = 0; i < nums.length; i++) {
        var srcNum = target - nums[i]
        var index = nums.indexOf(srcNum)

        if (index > -1 && index !== i) {
            twoKeys.push(i, index)
            break
        }
    }

    return twoKeys
};